src/components/LeftSidebar/SearchBox/SearchBox.vue

```diff
@@ -69,29 +71,20 @@ export default {
 			type: Boolean,
 			default: false,
 		},
+		disabled: {
```

Comment:
We don't use `disabled` prop anymore, so could be removed completely from component (it makes a little sense to keep it just in case)